test/integration_tests.jl

```diff
@@ -152,3 +168,22 @@
 
     _cleanup_old_branches(url)
 end
+
+@testset "$(service)" for service in [GITHUB, GITLAB]
+    personal_access_token = ENV["INTEGRATION_PAT_$(service)"]
+    test_repo = ENV["INTEGRATION_TEST_REPO_$(service)"]
+
+    api = get_api(service, personal_access_token)
+    user, _ = GitForge.get_user(api)
+    repo, _ = GitForge.get_repo(api, test_repo)
```

Comment:
This could maybe use `get_api_and_repo`?
https://github.com/JuliaRegistries/CompatHelper.jl/blob/v3-dev/src/utilities/ci.jl